### PR TITLE
feat: add more

### PR DESCRIPTION
**Pull request by JuanPablo2655**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+		<p>I added this paragraph in order to test the GitHub status push for the GitHub assignment.</p>
+		<p>I added this paragraph in order to test the GitHub status push for the GitHub assignment.</p>
+		<p>I added this paragraph in order to test the GitHub status push for the GitHub assignment.</p>
+		<p>I added this paragraph in order to test the GitHub status push for the GitHub assignment.</p>
+		<p>I added this paragraph in order to test the GitHub status push for the GitHub assignment.</p>
+		<p>I added this paragraph in order to test the GitHub status push for the GitHub assignment.</p>
+		<p>I added this paragraph in order to test the GitHub status push for the GitHub assignment.</p>
+		<p>I added this paragraph in order to test the GitHub status push for the GitHub assignment.</p>
+		<p>I added this paragraph in order to test the GitHub status push for the GitHub assignment.</p>
+		<p>I added this paragraph in order to test the GitHub status push for the GitHub assignment.</p>
```
Comment (lucnguy6912) [inline index.html]: Delete these, too many paragraphs

**Review by lucnguy6912 — CHANGES_REQUESTED** [review]
Hey brother there are too many Ps. you pushing P too much

File: index.html (hunk 1 above)
Comment (JuanPablo2655) [reply index.html]: oh my bad

**Review by lucnguy6912 — APPROVED** [review]
Looks sensational

**Pull request merged**